Added feedback page and wired it to the dashboard

fru1tstand opened the pull request.

fru1tstand committed:
Added feedback page and wired it to dashboard

jonathanpli commented on src/main/resources/templates/feedback.html:
```diff
@@ -0,0 +1,198 @@
+
+		<div class="eval-flag">
+			<div class="global-title-font">Grading Rubric</div>
+			<div class="eval-rubric">
```
Very small thing, but based off the comments provided for the code submission, was a 13.5 warranted? Wouldn't it be a higher score?

fru1tstand replied:
Masochistic grader. But either way, a final pass will be done to correct inconsistencies and improve flow (#44). Added this item onto that list.

fru1tstand committed:
Merge branch 'flow-finalize' into feedback

fru1tstand committed:
Merge branch 'flow-finalize' into feedback

fru1tstand committed:
First draft of earnings page

fru1tstand committed:
Changed layout of earnings and scaled down fonts and cards

fru1tstand committed:
Scaled down navbar sizes

fru1tstand committed:
Narrowed sidebar, scaled down fonts and spacing

fru1tstand committed:
Abstracted 'card' style; Added chartjs to dashboard; Scaled down dashboard

fru1tstand committed:
Fixed link colors on dashboard

fru1tstand committed:
Slimmed down sidebar for challenge, eval, and feedback; Scaled fonts,
spacing, and sizing to better fit; Abstracted review split screen to
code-split in layout scss file

fru1tstand committed:
Fixed li-sidebar active tab; Moved li-sidebar-nav-element from partial template to macro inside li-sidebar-page template

fru1tstand committed:
Removed page-specific partial templates in favor of macros

fru1tstand committed:
Reduced global nav link size

fru1tstand committed:
Push to server

fru1tstand committed:
Fixed box shadow layering

fru1tstand committed:
Fixed nav spacing; Code cleanup

fru1tstand committed:
Changed graph order to chronological from reverse chronological

fru1tstand committed:
Merge branch 'earnings' into quickfix-cleanups

fru1tstand committed:
Merge pull request #47 from DumplingLTD/quickfix-cleanups

Site-wide Resizing and Code Cleanup

fru1tstand committed:
Merge pull request #46 from DumplingLTD/earnings

Added Earnings Page

fru1tstand commented:
#42
